tpl/tplimpl/embedded/templates/_default/rss.xml

```diff
@@ -16,7 +16,7 @@
     <title>{{ if eq  .Title  .Site.Title }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{.}} on {{ end }}{{ .Site.Title }}{{ end }}</title>
     <link>{{ .Permalink }}</link>
     <description>Recent content {{ if ne  .Title  .Site.Title }}{{ with .Title }}in {{.}} {{ end }}{{ end }}on {{ .Site.Title }}</description>
-    <generator>Hugo -- gohugo.io</generator>{{ with .Site.LanguageCode }}
+    <generator>Gotham -- gothamhq.com</generator>{{ with .Site.LanguageCode }}
```

Comment:
This is not relevant to the meta generator tag and the bug we're fixing. This isn't even HTML. I suggest we put this piece into its own PR.

Also, the file above this, you manually changed one and the other was generated right? Did you do that on purpose?

Reply:
Yes, I will pull this out of this PR and do it separately.

No, I changed both. Will need to review both of these files in the separate PR.

Reply:
Gotcha. So we don't want to do that here. Files in the tpl path that end with `autogen.go` are automatically generated from the other tpl files. So never edit those directly.

If you edit something in tpl such as `tpl/tplimpl/embedded/templates/_default/rss.xml`, after you save the file (and any other tpl changes), you can run `mage generate` to run the Go generation script which will create the other files. Then commit those changes.

This process isn't documented as far as I know and I had to learn the hardway. I created an Issue for us to document this going forward: https://github.com/gothamhq/gotham/issues/58